packages/source-fetcher/lib/polyfill.ts

```diff
@@ -0,0 +1,12 @@
+// must polyfill AbortController to use axios >=0.20.0, <=0.27.2 on node <= v14.x
+import { AbortController, AbortSignal } from "node-abort-controller";
```

Comment:
3rd time, perhaps consider factoring this out to a utility module? Maybe this warrants a new truffle-package to handle these utility functions? Would be interested to hear what others think.

Reply:
So I don't know what is better here; 6 lines of code duplicated 4 times or 30 separate packages which each contain  6 
 lines of code that now need to be maintained for instances like this.

Reply:
I'd be opposed to a `@truffle/common` on the basis of the naming not really defining a well-defined scope (these are sometimes referred to as "god modules").

That said, I think a `@truffle/polyfills` that behaves like the module here (applies polyfills as a side-effect of module evaluation) is probably warranted.

This PR is currently blocking #5367. My request is that you guys allow this bit of cruft through on this PR, and then let me clean it up in #5367, as that PR will introduce a second polyfill, and personally I find it easier to structure modules like the one I'm proposing here (`@truffle/polyfills`) when I have more than a single concern to place on them.

If you guys agree, I'll add a note to #5367 with a link back to this convo so that it won't be missed on review of that one.

Reply:
That sounds like a good idea @benjamincburns.

Reply:
Note added. https://github.com/trufflesuite/truffle/pull/5367#issuecomment-1211540122